paket.dependencies

```diff
@@ -0,0 +1,32 @@
+storage: packages
+source https://nuget.org/api/v2
```

Comment:
You might consider changing this to `https://api.nuget.org/v3/index.json` (same in group Test) for better perf.

Reply:
@kerams, review appreciated. There is some additional work required, but core functionality is there. Also, for some reason I can not restore test db any more, - it fails with "pg_restore: [archiver] unsupported version (1.14) in file header" error. Update of postgres didn't help...

Reply:
I made the backup on 12.1 using pgAdmin. That could be the problem.